### DISCUSSION TITLE
联机什么时候复活

### DISCUSSION THREAD

**Discussion by AKsitaWa** [Meloong-Git/PCL#4038] [讨论]
都一年半了还没好吗

**Comment by allMagicNB** [comment]
# 你先别急
_至于联机，因为网易还在继续代理中国版，我和 HMCL 的维护者都觉得还得谨慎一点……况且现在也有 Mod 支持联机了……咳咳。——龙腾猫跃_

**Comment by SALTWOOD** [comment]
网易没死透之前基本没戏

**Reply by iPodtouch6557** [reply]
早点给我逝去，啥都没有封闭生态，就是在android和iOS对比下的iOS，还没有任何可以打的闪光点

**Reply by Pigeon0v0** [reply]
> 早点给我逝去，啥都没有封闭生态，就是在android和iOS对比下的iOS，还没有任何可以打的闪光点

别尬黑，iOS 比它开放

**Reply by SALTWOOD** [reply]
> 早点给我逝去，啥都没有封闭生态，就是在android和iOS对比下的iOS，还没有任何可以打的闪光点

别尬黑，喜欢归喜欢，不要黑不喜欢的

**Reply by iPodtouch6557** [reply]
> > 早点给我逝去，啥都没有封闭生态，就是在android和iOS对比下的iOS，还没有任何可以打的闪光点
> 
> 别尬黑，iOS 比它开放

首先，不要吵了
我觉得我有必要说明一下，让我说完
其次，我也不是黑iOS，就是有些资源iOS确实不好用，无法app侧载就是硬伤，即使iOS为了安全舍弃一点开放我倒是很支持。相反，我更喜欢iOS，好看好用，~只要手机能贴iOS我就买~

**Reply by SqrtTan** [reply]
> > > 早点给我逝去，啥都没有封闭生态，就是在android和iOS对比下的iOS，还没有任何可以打的闪光点
> > 
> > 
> > 别尬黑，iOS 比它开放
> 
> 首先，不要吵了 我觉得我有必要说明一下，让我说完 其次，我也不是黑iOS，就是有些资源iOS确实不好用，无法app侧载就是硬伤，即使iOS为了安全舍弃一点开放我倒是很支持。相反，我更喜欢iOS，好看好用，~只要手机能贴iOS我就买~

我一个同学天天研究ipa自签（

**Comment by Chiloven945** [comment]
嗯？联机？什么时候有这个功能的？
哦哦，想起来了

**Comment by Aloner-Studio** [comment]
在联机没有复活前，如果你想联机，可以尝试使用如开服器，联机侠，樱花内网穿透等与好友进行联机，另外，BakaXL启动器已支持联机
